jsonschema-core.xml

```diff
@@ -583,18 +583,16 @@
                         without requiring JSON Pointer references to be updated.
                     </t>
                     <t>
-                        To name subschemas in a JSON Schema document,
-                        subschemas can use "$id" to give themselves a document-local identifier.
-                        This is done by setting "$id" to a URI reference consisting
-                        only of a plain name fragment (not a JSON Pointer fragment).
-                        The fragment identifier MUST begin with a letter ([A-Za-z]), followed by
-                        any number of letters, digits ([0-9]), hyphens ("-"), underscores ("_"), colons
-                        (":"), or periods (".").
+                        To specify such a subschema identifier,
+                        the "$id" keyword is set to a URI reference with a name fragment (not a JSON Pointer fragment).
```

Comment:
I was using "plain name" rather than just "name" because that is the terminology used in the W3C fragment best practices document that we cite.  i think we should be consistent with that.